src/components/layout/Header.tsx

```diff
@@ -33,8 +33,8 @@ const Wrapper = styled.header`
   .connect-wallet {
     padding: 1rem;
     margin: 1rem;
-    border: 2px solid var(--color-btn-border);
-    color: var(--color-btn-border);
+    border: 2px solid #ff5097;
+    color: #ff5097;
```

Comment:
Why not use the colours from the global styles?

Reply:
I got tired of modifying in two places just for modifying a color of a scoped component
I think vars make sense if we think we will use them a lot.
I just left 

```
--color-text-primary: #3d414c;
--color-text-secondary: #8c8c8c;
```